content/helloworld-fromcftok8s.md

```diff
@@ -178,27 +178,29 @@ Now I can deploy this with
+Now my container is running on Kubernetes, but how do I talk to it? In Cloud Foundry, each app automatically gets a [route](https://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html), which is an external http endpoint that hits the default route specified by my app. If I don't specify it, CF assigns a so-called random route when I push my app for the first time. Every app in CF has a route, unless you actively make sure it doesn't get one. Yes, there are cases where that makes sense but that's off topic for now. 
 
-In Kubernetes, things are a bit more complicated. The Kubernetes deployment resource I used above doesn't care about making things reachable from the outside world - it cares about deploying stuff. As a result, the port exposed by the container (the one I specified in the Dockerfile) is only reachable from *within* the cluster - other containers and other pods can (theoretically) use it to talk to my app, but nobody else. Is is however not a good idea to do it this way either, since pods are ephemeral - they come and go. One key assumption Kubernetes makes about containers is that they die and have to be re-created all the time - as we all know, developers can't be trusted, right? So the deployment's main concern is to make sure that actual state of the deployment matches the wanted state expressed by it's yaml specification. If a container dies, it will recreate it (well, in fact that's what the underlying ReplicaSet does but hey). 
+In Kubernetes, things are a bit more complicated. The Kubernetes deployment resource I used above doesn't care about making things reachable from the outside world - it cares about deploying stuff and making sure that the actual state of the deployment matches the wanted state expressed by it's yaml specification. Nothing else. As a result, the port exposed by the container (the one I specified in the Dockerfile) is only reachable from *within* the cluster - other containers and other pods can (theoretically) use it to talk to my app, but nobody else. 
 
-Fair enough, so what else have we got? 
+Btw, a good article that helped me wrap my head around all this is ["Kubernetes Ingress for Beginners"](https://thenewstack.io/kubernetes-ingress-for-beginners/) by Nick Ramirez. Thanks Nick!
+
+As an aside, it is not a good idea to do it this way either, since pods are ephemeral - they come and go. One key assumption Kubernetes makes about containers is that they die and have to be re-created all the time - as we all know, developers can't be trusted, right? o the deployment's main concern is to make sure that the actual state of the deployment matches the wanted state expressed by its yaml specification. If a container dies, it will recreate it (well, in fact that's what the underlying ReplicaSet does but hey).
```

Comment:
```suggestion
As an aside, it is not a good idea to do it this way either, since pods are ephemeral - they come and go. One key assumption Kubernetes makes about containers is that they die and have to be re-created all the time - as we all know, developers can't be trusted, right? The deployment's main concern is to make sure that the actual state of the deployment matches the wanted state expressed by its yaml specification. If a container dies, it will re-create it  (well, in fact, that's what the underlying ReplicaSet does, but hey).
```